src/components/__tests__/StarRating.spec.ts

```diff
@@ -26,9 +28,9 @@ describe("StarRating", () => {
         expect(starRating).toBeElement(
```

Comment:
Please add new test case for different font size and one for custom font size

Reply:
Done